### PR TITLE
[DDW-857][DEBT] Integrate Chromatic

### PR DESCRIPTION
**Pull request by renanvalentin**
This PR integrates Chromatic to Daedalus development process.

## Todos

- [ ] Todo

## Screenshots

- Screenshot

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1638995113390700)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+on: push
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12"
```
Comment (przemyslaw-wlodek) [inline .github/workflows/chromatic.yml:11]: Should it be aligned with `"devEngines"` from our `package.json`?

**Review by przemyslaw-wlodek — CHANGES_REQUESTED** [review]

**Review by danielmain — APPROVED** [review]
I like that you improved the code, even if it was not necessary to this PR. Just we need to inform or QA colleagues to test the part of the code you touched (PublicKeyQRCodeDialog).

File: source/renderer/app/components/wallet/settings/ICOPublicKeyQRCodeDialog.js [hunk 2]
```diff
@@ -11,14 +11,14 @@ import Dialog from '../../widgets/Dialog';
 import iconCopy from '../../../assets/images/clipboard-ic.inline.svg';
 import styles from './PublicKeyQRCodeDialog.scss';
 import globalMessages from '../../../i18n/global-messages';
-import type { ReactIntlMessage } from '../../../types/i18nTypes';
+
+import { messages } from './ICOPublicKeyQRCodeDialog.messages';
```
Comment (renanvalentin) [inline source/renderer/app/components/wallet/settings/ICOPublicKeyQRCodeDialog.js:15]: The storybook book related to `ICOPublicKeyQRCodeDialog` is broken on develop branch:
https://www.chromatic.com/test?appId=61afa6b2acfbf7003ae248a3&id=61afab78e0748c003abdf55b

Comment (danielmain) [reply source/renderer/app/components/wallet/settings/ICOPublicKeyQRCodeDialog.js:15]: Thank you, @renanvalentin, you fixed that in this branch!

@dmitrii-gaico would you pls just check the ICO PublicKey QR Code dialog is still working? thank you :)

**Review by dmitrii-gaico — APPROVED** [review]
ICO Public key dialog looks good. Not affected. Thank you!

File: .github/workflows/chromatic.yml (hunk 1 above)
Comment (danielmain) [reply .github/workflows/chromatic.yml]: No, forget what package.json=>devEngines is defined it is just decoration. The only what counts is https://github.com/input-output-hk/daedalus/blob/66f9ea0cd87e44fc58801d5dc9d0542f4e24cc42/default.nix#L58 🤷‍

**Pull request merged**